tests/data_context/test_data_context_variables.py

```diff
@@ -578,7 +582,13 @@ def test_cloud_enabled_data_context_variables_e2e(
     new_site_name = f"docs_site_{''.join(random.choice(string.ascii_letters + string.digits) for _ in range(8))}"
     updated_data_docs_sites[new_site_name] = {}
 
-    context = DataContext(ge_cloud_mode=True)
+    # Call with explicit args the first time
+    context = DataContext(
```

Comment:
Do you want to unset the environment variables in a try/finally block for this test?